### ISSUE TITLE
[0.4] some receptor fingerprint cannot print `neurotransmitters`

### ISSUE THREAD

**Issue by xgui3783** [FZJ-INM1-BDA/siibra-python#247]
see https://github.com/FZJ-INM1-BDA/siibra-python/actions/runs/3969041796/jobs/6802998498

to reproduce:

```python
from siibra.features.molecular.receptor_density_fingerprint import ReceptorDensityFingerprint

all_features = ReceptorDensityFingerprint.get_instances()

for feature in all_features:
    assert isinstance(feat, ReceptorDensityFingerprint)
    print(feat.name, feat.id)
    print(feat.unit)
    print(feat.neurotransmitters)
    print(feat.data)

```

issue narrowed down to ill(?) formed tsv

e.g. preconf file at [features/fingerprints/receptor/f61c27af-d18e-4abc-9b02-96c09d49e542_CA1_fp_20200217_RAT_CORNU_AMMONIS_1_V3.json](https://jugit.fz-juelich.de/t.dickscheid/brainscapes-configurations/-/blob/master/features/fingerprints/receptor/f61c27af-d18e-4abc-9b02-96c09d49e542_CA1_fp_20200217_RAT_CORNU_AMMONIS_1_V3.json) , tsv file contained within is malformed:

```sh
curl 'https://object.cscs.ch/v1/AUTH_227176556f3c4bb38df9feea4b91200c/hbp-d000045_receptors-rat-CA1_pub/v2.0/CA1_fp_20200217.tsv'
```
returns
```tsv
"receptor label (CA1)"  "density (mean)"        "density (sd)"  "density (unit)"
"AMPA"  4199    350     "fmol/mg"
"kainate"       669     51      "fmol/mg"
"NMDA"  5290    318     "fmol/mg"
"GABAA" 1254    100     "fmol/mg"
"M1"    1530    73      "fmol/mg"
"M2"    333     49      "fmol/mg"
"alpha1"        223     19      "fmol/mg"
"alpha2"        430     51      "fmol/mg"
"5-HT1A"        811     168     "fmol/mg"
"5-HT2" 34      15      "fmol/mg"
```

pandas will interpret the keys to be e.g. `"AMPA"` rather than expected `AMPA`, leading to key error

**Comment by dickscheid** [comment]
Should be fixed in 07788bd5e3b7

**Comment by dickscheid** [comment]
@AhmetNSimsek can you confirm and close the issue?
